packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -677,8 +722,10 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
                 case WorkflowActions.EMAIL_ATTENDEE:
                   sendTo = bookingInfo.attendees.map((attendee) => attendee.email);
                   break;
-                /*case WorkflowActions.EMAIL_ADDRESS:
-                      sendTo = step.sendTo || "";*/
+                case WorkflowActions.EMAIL_ADDRESS:
+                  await verifyEmailSender(newStep.sendTo || "", user.id, userWorkflow.teamId, ctx.prisma);
```

Comment:
for added steps we only call it when we have bookings that need reminders scheduled. We should do it the same as we do for the edited steps

Reply:
Yeah removed it from here and added when creating new step